lib/alchemy/admin/locale.rb

```diff
@@ -46,15 +46,16 @@ def available_locale
       def locale_from_user
         return if !current_alchemy_user
         if user_has_preferred_language?
-          current_alchemy_user.language
+          current_alchemy_user.language.to_sym
```

Comment:
Cannot check at the moment as I am only on mobile but as far as I can see to_sym is only called in the detect for the condition but not applied to the actual locale. I will have a look tomorrow at this again.